doc/man3/SSL_set_quic_tls_cbs.pod

```diff
@@ -70,6 +70,11 @@ given SSL object I<s>, a set of callbacks are supplied in an B<OSSL_DISPATCH>
 table via I<qtdis>. The I<arg> parameter will be passed as an argument when the
 various callbacks are called.
 
+All callbacks run in context of SSL_do_handshake() and SSL_read() (including
+SSL_read_ex, SSL_peek, SSL_peek_ex). After SSL handshake is confirmed the
+QUIC stack should call any SSL_read() variant whenever CRYPTO frame is available
```

Comment:
What about the zero length argument to SSL_read()? Should that be explicitly mentioned?